shared/common-adapters/user-proofs.desktop.js

```diff
@@ -76,18 +76,18 @@ export default class ProofsRender extends Component<void, Props, void> {
   }
 
   render () {
-    const {isLoading} = this.props
+    const {loading} = this.props
```

Comment:
Agreed. I'll be squashing all of these. I have a couple low-value commits like this that will all be squished together.